src/traces/parcoords/parcoords.js

```diff
@@ -579,7 +583,9 @@ module.exports = function(gd, svg, parcoordsLineLayers, cdModule, layout, callba
                     .tickValues(d.ordinal ? // and this works for ordinal scales
                         sdom :
                         null)
-                    .tickFormat(d.ordinal ? function(d) { return d; } : function(d) { return linearFormat(d); })
+                    .tickFormat(function(v) {
+                        return d.ordinal ? v : linearFormat(v, d.tickFormat);
```

Comment:
@archmoj I'm a little concerned with the image diff for `gl2d_parcoords_rgba_colorscale`

![Peek 2019-06-17 10-31](https://user-images.githubusercontent.com/6675409/59612446-18d77b00-90eb-11e9-97ce-1f76c15ea421.gif)

I don't understand why those `m` suffixes are now showing up.

Reply:
Revised in https://github.com/plotly/plotly.js/pull/3966/commits/11b94340b7138510e4ee9f345e91c40cf795b826.

Reply:
Hmm. This now shows up as:

![image](https://user-images.githubusercontent.com/6675409/59775378-1c9c0680-927f-11e9-8c57-482575206c45.png)

this seems odd as the `values` are:

![image](https://user-images.githubusercontent.com/6675409/59775433-3b9a9880-927f-11e9-97ba-b74c12098350.png)

is that expected with the parcoords `.3s` `tickformat` default?

Reply:
The default is `3s` not `.3s` https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/traces/parcoords/attributes.js#L56-L58
Does that make sense?

Reply:
Ah, well there's nothing wrong with your code:

![image](https://user-images.githubusercontent.com/6675409/59777376-91247480-9282-11e9-9c38-d6c290f3d5f6.png)

(using http://bl.ocks.org/zanarmstrong/05c1e95bf7aa16c4768e)

I'm not sure what the `m` stands for here though.

----------

Maybe it's time to ditch the (bad) `tickformat` default altogether.

@arbscht can you generate try re-generating all the parcoords baselines w/o that `tickformat` default on a separate branch?

Reply:
m==milli - but yeah, we shouldn't use that by default - `0.003` is much better than `3m`. If we use the `Axes.tickText` default formatting it skips milli but does use all the other SI prefixes.

Reply:
Revised in https://github.com/plotly/plotly.js/pull/3966/commits/a80bf49957e08754934c2c1edeeb634e1244ad67.